src/ontology/foodon-edit.owl

```diff
@@ -42474,6 +42547,18 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_03310934 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03310934">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03302012"/>
+        <obo:IAO_0000115 xml:lang="en">A smoked sausage made of finely minced turkey meat and served in a bread roll.</obo:IAO_0000115>
```

Comment:
a turkey frank isn't necessarily served in a bread roll, so take that part out of the definition; and same comments as for chicken frankfurter.